zenodo/modules/fixtures/data/funders.json

```diff
@@ -11,7 +11,7 @@
     },
     "name": "National Health and Medical Research Council",
     "parent": {
-      "$ref": "http://dx.doi.org/10.13039/501100003921"
+      "$ref": "https://doi.org/10.13039/501100003921"
```

Comment:
We resolve those DOIs "internally" with: https://github.com/inveniosoftware/invenio-openaire/blob/master/invenio_openaire/resolvers/funders.py#L44-L45

which should support both dx.doi.org and doi.org, but just in case this fails in a test somewhere, it might require some fixes upstream.